filer/utils/checktrees.py

```diff
@@ -1,4 +1,5 @@
 from django.db.models import Count
+from filer.models import Folder
```

Comment:
Where do we use this import?

Reply:
If we do need this import, I think the correct package is `from filer.models.foldermodels import Folder`

Reply:
Not used. I'll remove it.